esp-hal-embassy/src/time_driver.rs

```diff
@@ -12,7 +12,7 @@ use esp_hal::{
 
 pub const MAX_SUPPORTED_ALARM_COUNT: usize = 7;
 
-pub type Timer = OneShotTimer<'static, ErasedTimer>;
+pub type Timer = OneShotTimer<ErasedTimer>;
```

Comment:
Please undo this change

Reply:
fair point, completely missed that - whats the best way to pin versions while testing? would it be to fork from the v0.19.0 tag or just check these things manually?